settings.json

```diff
@@ -1,753 +1,973 @@
 {
-    "pageTemplates": "WebModeler",
-    "cssFiles": [
-        "styles/web/css/main.css"
+    "Widget": [
+        {
+            "name": "Spacing top",
+            "type": "Dropdown",
+            "description": "The spacing above this element.",
+            "options": [
+                {
+                    "name": "Outer none",
```

Comment:
Change names of existing margin design properties to "Outer <previous name>" and introduce padding design properties (e.g. "Inner small")